packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.js

```diff
@@ -101,6 +101,15 @@ export default class QuanticTab extends LightningElement {
   }
 
   get tabClass() {
-    return `slds-tabs_default__item ${this.isActive ? 'slds-is-active' : ''}`;
+    return `slds-tabs_default__item slds-tabs_button slds-button ${
+      this.isActive ? 'slds-is-active' : ''
+    }`;
+  }
+  get isPressed() {
+    return `${this.isActive}`;
+  }
+
+  get arialLabelText() {
+    return `${this.label}`;
   }
```

Comment:
I think I would remove that function for now and just use `this.label` in the HTML, see my other comment in the html file

Reply:
arialLabelText removed